tests/Application/Actions/StatusTest.php

```diff
@@ -85,11 +87,11 @@ public function testMissingDoctrineORMProxy(): void
     }
 
     private function getConnectedMockEntityManager(
-        string $proxyPath = '/var/www/html/var/doctrine/proxies',
+        string $proxyPath = __DIR__ . '/../../../var/doctrine/proxies',
```

Comment:
Any reason we needed to change this? Wasn't it working fine before? Plus it was more legible?

Reply:
This makes a it a bit more flexible to run in different places - specifically this way it can run on my machine outside docker, where the path is not `/var/www/html/var/doctrine/proxies` but `/home/barney/projects/matchbot/var/doctrine/proxies`

Reply:
@alihejazi Yes, it was runnable through docker, just thought it was worth making this change to allow running it both ways.